Code_experimental_design.py

```diff
@@ -1,5 +1,6 @@
 """This package contains the code for the experimental design. It includes all trials step by step.
```

Comment:
Common style is to set """ and then Enter so that all info is between """

Example:
"""
This the documentation
"""